onnx/reference/reference_evaluator.py

```diff
@@ -51,6 +54,12 @@ class ReferenceEvaluator:
         kernels are added in `new_ops` and are used instead of the
         inner implementation if list *new_ops* does not already contain
         one.
+    :param save_intermediate: None or a folder, if not empty,
+        this folder is used to save every intermediate inputs and
+        outputs of every node in the main graph,
+        following the same organization of the backend test,
+        it can be used later to compare intermediate outputs
+        or two different runtimes.
```

Comment:
nit: clarify how the sub-folders are named? I see it is `f"test_node_{index}_{node.op_type}"`